list_people.rb

```diff
@@ -0,0 +1,16 @@
+def list_people(people)
+  puts 'All people:'
+
+  if people.length.positive?
+
+    people.each do |person|
+      if person.respond_to?('specialization')
+        puts "[TEACHER] Name: #{person.name.capitalize}, ID: #{person.id}, Age: #{person.age} }"
+      else
+        puts "[STUDENT] Name: #{person.name.capitalize}, ID: #{person.id}, Age: #{person.age} }"
+      end
+    end
+  else
+    puts 'No people in the school, create some.'
+  end
```

Comment:
Great job @Ezzywealth 🤝

- [ ] The app tends to break whenever I try to create a person. However, you have done great in implementing the requested changes by the previous reviewer.
![person](https://user-images.githubusercontent.com/63917725/231820010-8dab67e8-b810-4421-ac1b-4f0f5f689569.PNG)